RVFitter/RVFitter.py

```diff
@@ -1427,6 +1432,12 @@ def plot_fits_and_residuals(self,
         labels = [this_fitter.label for this_fitter in self.list_of_fitters]
         fig.legend(handles, labels, ncol=2, loc='lower center')
 
+        filename = "fits_and_residuals.png"
```

Comment:
So the idea not to give the filename as an argument to the function is that the outputfolder will determine "where it belongs"? There is no use-case for the user to rename this figure into something else?
Maybe make a default argument such as None which then falls back to what you suggest. Then the user can also choose to name the file otherwise.